content/consul/v1.21.x/content/docs/secure/sso/auth0.mdx

```diff
@@ -7,86 +7,28 @@ description: >-
 
 <Note title="Enterprise Only">
 
  The single sign-on (SSO) functionality demonstrated here requires self-managed [Consul Enterprise](https://www.hashicorp.com/products/consul/pricing/). If you've purchased or wish to try out Consul Enterprise, refer to [how to access Consul Enterprise](/consul/docs/enterprise#access-consul-enterprise).
 
 </Note>
 
-## Overview
-
-Consul Enterprise 1.8.0 introduced the ability to configure single sign-on (SSO) and to authenticate with Consul using [OIDC](https://en.wikipedia.org/wiki/OpenID_Connect).
-
-OIDC authentication is useful when you want to deploy SSO widely in your organization and do not want to manage access with Consul ACL tokens.
-
-Once implemented, SSO will enable an interactive login procedure that can be initiated from either the Consul UI or the command line.
-
-In this tutorial you will configure [Auth0](https://auth0.com/) as an identity provider. Your configuration will use the user metadata in Auth0 to automatically grant permissions in Consul ACL.
-
-This is a multi-step process that includes:
-
-- configure an application in Auth0 for Consul integration;
-- set up Consul ACL policies and roles to provide permissions to users
-- configure user metadata and use binding rules to automatically assign permissions to users based on their metadata values.
-
-This approach unifies the user permissions grant without the need to create individual user tokens or to define permissions individually for the different users. The tutorial also shows one example of how to define service permissions grants for a user based on unique metadata values of the individual user, making this approach also suitable for finer grained permissions.
-
-## Prerequisites
-
-In order to complete this tutorial the following prerequisites need to be satisfied:
-
-- A Consul Enterprise datacenter or local dev agent. The datacenter will need ACLs enabled to use the OIDC authentication method. You can use the following ACL configuration as a minimal example to follow the steps in this tutorial:
-
-<CodeTabs heading="Consul agent configuration">
-
-```hcl
-acl {
- enabled        = true
- default_policy = "deny"
-
- tokens {
-   initial_management = "root"
-   agent  = "root"
- }
-}
-```
-
-```json
-{
-  "acl": {
-    "enabled": true,
-    "default_policy": "deny",
-    "tokens": {
-      "initial_management": "root",
-      "agent": "root"
-    }
-  }
-}
-```
+## Workflow
 
-</CodeTabs>
-
-
-- A valid [Auth0](https://auth0.com/) account. If you do not have one, create 
-one now, before continuing with the tutorial. The tutorial will show you the 
-configuration steps necessary to configure your Auth0 account to interact with 
-Consul.
+1. [Configure an Auth0 application for Consul integration](#configure-auth0-application)
+1. [Create Auth0 users and metadata](#create-auth0-users-and-metadata)
```

Comment:
```suggestion
1. [Create a rule to associate Auth0 users and Consul metadata](#create-auth0-users-and-metadata)
```
More descriptive for what the step entails, assuming a user already has Auth0 users to integrate.